tchannel/tornado/tchannel.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, name, hostport=None, process_name=None,
         else:
             self._handler = dispatcher
 
-        self.peers = PeerGroup(self)
+        self.peer_group = PeerGroup(self)
```

Comment:
Changing the name is because the ``PeerGroup`` class also has the ``peers`` property. If we want to use it, it will look like ``tchannel.peers.peers`` which doesn't make sense. So I change the name.